BasicTest_dnxcore50/project.json

```diff
@@ -1,32 +1,31 @@
 {
   "version": "1.0.0-*",
   "description": "StackExchange.Redis.BasicTest dnxcore50",
-  "authors": [ "" ],
-  "tags": [ "" ],
-  "projectUrl": "",
-  "licenseUrl": "",
-
-  "compile": [
-    "../BasicTest/Program.cs"
-  ],
+  "buildOptions": {
+    "compile": {
+      "include": [
+        "../BasicTest/Program.cs"
+      ]
+    }
+  },
   "dependencies": {
     "StackExchange.Redis": {
-      "version": "1.1.572-alpha",
+      "version": "1.1.*",
       "target": "project"
     }
   },
   "commands": {
     "run": "BasicTest_dnxcore50"
   },
-
-  
   "frameworks": {
-    "dnxcore50": {
+    "netcoreapp1.0": {
+      "buildOptions": {
+        "define": [ "CORE_CLR" ]
```

Comment:
Yep - thanks! I'm aware of that one, but the define has also changed every time we re-visit it through the pre-releases. For the moment, I think it's better to opt for our self-declared defines until RTM locks them in (to eliminate some git churn).